modelon/impact/client/entities/case.py

```diff
@@ -28,9 +29,17 @@ class _CaseRunInfo:
     Class containing Case run info.
     """
 
-    def __init__(self, status: CaseStatus, consistent: bool):
+    def __init__(
+        self,
+        status: CaseStatus,
+        consistent: bool,
+        datetime_started: int,
+        datetime_finished: int,
```

Comment:
It would be better if a `datetime` was sent in instead of `int`.

Reply:
Also, I think there are cases where `datetime_finished` is None? For example if the case has not finished running. Not sure about `datetime_started` and if it always have to be set.

Reply:
Good catch! I looked up the code(see below) and it looks like datetime_started couldn't be none, but datetime_finished could. I have now added checks for both to not get errors and a logger message in case any of them is None.
```
class CaseRunInfo:
    def __init__(
        self,
        status=CaseStatus.NOT_STARTED,
        datetime_started=None,
        datetime_finished=None,
        failed_at=CaseFailedAt.NONE,
        consistent=False,
    ):
        self.status = status
        self.datetime_started = datetime_started or datetime.datetime.utcnow()
        self.datetime_finished = datetime_finished
```